apheleia.el

```diff
@@ -583,48 +666,29 @@ even if a formatter is configured."
     "Apheleia does not support remote files"))
 
 ;;;###autoload
-(defun apheleia-format-buffer (command &optional callback)
+(defun apheleia-format-buffer (commands &optional callback)
   "Run code formatter asynchronously on current buffer, preserving point.
 
-Interactively, run the currently configured formatter (see
+COMMANDS is a list of the values from `apheleia-formatters'. If
+called interactively, run the currently configured formatters (see
 `apheleia-formatter' and `apheleia-mode-alist'), or prompt from
 `apheleia-formatters' if there is none configured for the current
 buffer. With a prefix argument, prompt always.
 
-In Lisp code, COMMAND is similar to what you pass to
-`make-process', except as follows. Normally, the contents of the
-current buffer are passed to the command on stdin, and the output
-is read from stdout. However, if you use the symbol `file' as one
-of the elements of COMMAND, then the filename of the current
-buffer is substituted for it. (Use `filepath' instead of `file'
-if you need the filename of the current buffer, but you still
-want its contents to be passed on stdin.) If you instead use the
-symbol `input' as one of the elements of COMMAND, then the
-contents of the current buffer are written to a temporary file
-and its name is substituted for `input'. Also, if you use the
-symbol `output' as one of the elements of COMMAND, then it is
-substituted with the name of a temporary file. In that case, it
-is expected that the command writes to that file, and the file is
-then read into an Emacs buffer. Finally, if you use the symbol
-`npx' as one of the elements of COMMAND, then the first string
-element of COMMAND is resolved inside node_modules/.bin if such a
-directory exists anywhere above the current `default-directory'.
-
-In any case, after the formatter finishes running, the diff
-utility is invoked to determine what changes it made. That diff
-is then used to apply the formatter's changes to the current
-buffer without moving point or changing the scroll position in
-any window displaying the buffer. If the buffer has been modified
-since the formatter started running, however, the operation is
-aborted.
```

Comment:
It would be nice if in this docstring we had a pointer saying that the format is explained in the docstring of `apheleia-formatters`.

Reply:
I thought the docstring near the start of the function did that. Or would you like it to be a little more descriptive?

```
COMMANDS is a list of the values from `apheleia-formatters'. 
```